README.rst

```diff
@@ -1,3 +1,18 @@
+THIS PROJECT IS NO LONGER SUPPORTED
+======================================
+
+This project relies on the iGeom interface to provide access to an
+underlying CAD engine.  That interface is no longer actively supported
+in any project and none of this historical implementations continue to
+be valid due to licensing and other restrictions.
+
+In theory, it should be possible to revive this by providing access to
+an open source CAD engine, such as OpenCascade (OCC), either by implementing
+a lightweight version of iGeom for OCC or modifying this code to directly
+access OCC.  The permissive license of this should allow others to tackle
+this if/when they have interest.
+
+Below is the original README file describing other aspects of this package.
```

Comment:
It's probably worth mentioning that this capability is available natively with Coreform Cubit, similar to what was mentioned in https://github.com/svalinn/Cubit-plugin/pull/160.